docs/topics/http-api-nodejs-package-json.adoc

```diff
@@ -6,35 +6,69 @@
 ----
 {
   "name": "nodejs-rest-http",
-  "version": "0.0.0",
-  "private": true,
+  "version": "1.1.0",
+  "author": "Red Hat, Inc.",
+  "license": "Apache-2.0",
   "scripts": {
-    "start": "PORT=8080 node ./bin/www", <1>
-    "openshift": "nodeshift --osc.strictSSL=false", <2>
-    "test": "tape test/*-test.js" <3>
+    "test": "tape test/*.js | tap-spec", <1>
+    "lint": "eslint test/*.js app.js bin/*",
+    "prepare": "nsp check",
+    "coverage": "nyc npm test",
+    "coveralls": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
+    "ci": "npm run lint && npm run coveralls",
+    "dependencyCheck": "szero . --ci",
+    "release": "standard-version",
+    "openshift": "nodeshift --strictSSL=false --nodeVersion=8.x", <2>
+    "postinstall": "license-reporter report && license-reporter save --xml licenses.xml",
+    "start": "PORT=8080 node ./bin/www" <3>
   },
-  "files": [ <4>
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bucharest-gold/nodejs-rest-http.git"
+  },
+  "files": [  <4>
     "package.json",
     "app.js",
     "public",
-    "bin"
+    "bin",
+    "LICENSE",
+    "licenses"
   ],
-  "dependencies": { <5>
-    "body-parser": "~1.17.1",
-    "debug": "~2.6.3",
-    "express": "^4.15.3"
+  "bugs": {
+    "url": "https://github.com/bucharest-gold/nodejs-rest-http/issues"
   },
-  "devDependencies": { <6>
-    "nodeshift": "^0.3.0",
+  "homepage": "https://github.com/bucharest-gold/nodejs-rest-http",
+  "devDependencies": {  <5>
+    "coveralls": "^3.0.0",
+    "eslint": "~4.17.0",
+    "eslint-config-semistandard": "~12.0.0",
+    "eslint-config-standard": "~11.0.0-beta.0",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-node": "^6.0.0",
+    "eslint-plugin-promise": "~3.6.0",
+    "eslint-plugin-react": "~7.6.0",
+    "eslint-plugin-standard": "~3.0.1",
+    "license-reporter": "^1.1.3",
+    "nodeshift": "^1.3.0",
+    "nsp": "~3.1.0",
+    "nyc": "~11.4.1",
+    "standard-version": "^4.2.0",
     "supertest": "^3.0.0",
-    "tape": "^4.6.3"
+    "szero": "^1.0.0",
+    "tap-spec": "~4.1.1",
+    "tape": "~4.8.0"
+  },
+  "dependencies": { <6>
+    "body-parser": "^1.18.2",
+    "debug": "^3.1.0",
+    "express": "^4.16.0"
   }
 }
 ----
 
-<1> A `npm` script for starting this application.  Run with `npm start`.
+<1> A `npm` script for running unit tests.  Run with `npm run test`.
 <2> A `npm` script for deploying this application to {OpenShiftLocal}.  Run with `npm run openshift`.  The `osc.strictSSL` option allows us to deploy to {OpenShiftLocal} instances with self-sign certificates.
```

Comment:
It should be "self-signed certificates".

Reply:
nice catch!